.github/workflows/platformDeploy.yml

```diff
@@ -85,31 +85,19 @@ jobs:
           MYAPP_UPLOAD_KEY_PASSWORD: ${{ secrets.MYAPP_UPLOAD_KEY_PASSWORD }}
           VERSION: ${{ env.VERSION_CODE }}
 
-      - name: Archive Android sourcemaps
-        uses: actions/upload-artifact@v4
-        with:
-          name: android-sourcemap-${{ github.event.release.tag_name }}
-          path: android/app/build/generated/sourcemaps/react/productionRelease/index.android.bundle.map
-
-      - name: Upload Android build to GitHub artifacts
-        if: ${{ !fromJSON(env.SHOULD_DEPLOY_PRODUCTION) }}
-        uses: actions/upload-artifact@v4
-        with:
-          name: app-production-release.aab
-          path: android/app/build/outputs/bundle/productionRelease/app-production-release.aab
-
       - name: Upload Android build to Browser Stack
         if: ${{ !fromJSON(env.SHOULD_DEPLOY_PRODUCTION) }}
         run: curl -u "$BROWSERSTACK" -X POST "https://api-cloud.browserstack.com/app-live/upload" -F "file=@./android/app/build/outputs/bundle/productionRelease/app-production-release.aab"
         env:
           BROWSERSTACK: ${{ secrets.BROWSERSTACK }}
 
+      - name: Upload Android sourcemaps to GitHub Release
+        if: ${{ !fromJSON(env.SHOULD_DEPLOY_PRODUCTION) }}
```

Comment:
@roryabraham We should only upload Android/iOS on staging, right?

Reply:
yeah